emcc.py

```diff
@@ -856,6 +856,11 @@ def get_cflags(options, user_args):
              '-D__unix',
              '-D__unix__']
 
+  # When wasm EH is enabled, we use the legacy pass manager because the new pass
+  # manager + wasm EH has some known bugs. TODO Use the new pass manager.
```

Comment:
when adding TODOs tracked by bugs, it's a good idea to add a link to the bug along with the todo.

Reply:
Done